iidm/iidm-modification/src/main/java/com/powsybl/iidm/modification/scalable/ScalingParameters.java

```diff
@@ -39,16 +42,15 @@ public enum Priority {
     public static final boolean DEFAULT_ALLOWS_GENERATOR_OUT_OF_ACTIVE_POWER_LIMITS = false;
     public static final Priority DEFAULT_PRIORITY = ONESHOT;
     public static final ScalingType DEFAULT_SCALING_TYPE = DELTA_P;
+    public static final Set<String> DEFAULT_IGNORED_INJECTION_IDS = Collections.emptySet();
 
     private Scalable.ScalingConvention scalingConvention = DEFAULT_SCALING_CONVENTION;
-
     private boolean reconnect = DEFAULT_RECONNECT;
-
     private boolean constantPowerFactor = DEFAULT_CONSTANT_POWER_FACTOR;
-
     private boolean allowsGeneratorOutOfActivePowerLimits = DEFAULT_ALLOWS_GENERATOR_OUT_OF_ACTIVE_POWER_LIMITS;
     private ScalingType scalingType = DEFAULT_SCALING_TYPE;
     private Priority priority = DEFAULT_PRIORITY;
+    private Set<String> ignoredInjectionIds = DEFAULT_IGNORED_INJECTION_IDS;
```

Comment:
This parameter should also be settable via the config file (see method `load()`) and a JSON file (see the content of the `...scalable.json` package).

Reply:
Updated the load method and the json serializers/deserializers.